example/src/test/test2.entry.js

```diff
@@ -0,0 +1,3 @@
+import shared from '/partials/shared'
```

Comment:
I think you get it by now

Reply:
I like it better with a leading slash

On top of that, it keeps things consistent if we have a directory in the root of the src folder that also happens to be in the node_modules directory.

Have you checked the contents of that directory? It currently contains 964 'reserved' words.